api/src/main/java/io/minio/MakeBucketArgs.java

```diff
@@ -16,19 +16,15 @@
 
 package io.minio;
 
-public final class MakeBucketArgs {
-  private final Bucket bucket;
+public final class MakeBucketArgs extends BucketArgs {
   private final boolean objectLock;
 
   MakeBucketArgs(Builder builder) {
-    this.bucket = builder.bucket;
+    this.name = builder.name;
+    this.region = builder.region;
```

Comment:
`BucketArgs` already has these. You would need to make them `protected` in `BucketArgs`